modules/virt-customizing-storage-profile.adoc

```diff
@@ -31,21 +32,7 @@ If you create a data volume and omit YAML attributes and these attributes are no
 $ oc edit storageprofile <storage_class>
 ----
 +
-.Example storage profile
-[source,yaml]
-----
-apiVersion: cdi.kubevirt.io/v1beta1
-kind: StorageProfile
-metadata:
-  name: <unknown_provisioner_class>
-# ...
-spec: {}
-status:
-  provisioner: <unknown_provisioner>
-  storageClass: <unknown_provisioner_class>
-----
-+
-. Provide the needed attribute values in the storage profile:
+. Specify the *accessModes* and *volumeMode* values you want to configure for the storage profile, as shown in the following example:
```

Comment:
So the example of how to edit the accessModes and volumeMode was now moved to `virt-customizing-storage-profile-default-cloning-strategy.adoc` module... I think we should either combine them or split into different modules

Reply:
I didn't change the `virt-customizing-storage-profile-default-cloning-strategy.adoc` module as part of this bug fix, so what I would like to do is readd how to how to edit the accessModes and volumeMode to the `virt-customizing-storage-profile module`, as it was.